_gsocproposals/2020/proposal_IntAlertSystem.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Intelligent Alert Management System For HEP experiments
+layout: gsoc_proposal
+project: CMS-Monitoring
```

Comment:
the project name has to match the one in the project file, so it should read "project: CMS". Otherwise the links in the page are not made properly.